### PR TITLE
Change poise-python to pyenv

### PR DESCRIPTION
**Pull request by JustasUmbrasas**

### CODE REVIEW

**Comment by ton31337** [comment]
Tested?

**Comment by JustasUmbrasas** [comment]
Tested without exazk. Exazk doesn't work for unrelated reasons.

**Review by ton31337 — CHANGES_REQUESTED** [review]
Increase version in metadata.rb

File: metadata.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@
 license          'Apache 2.0'
 description      'Installs/Configures exabgp'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.1.19'
+version          '0.1.20'
```
Comment (ton31337) [inline metadata.rb:7]: Can you change to 0.2.0 to mark as a more important change?

**Review by ton31337 — APPROVED** [review]

**Pull request merged**